### ISSUE TITLE
RTKPLOT--Map image

### ISSUE THREAD

**Issue by Leungholammmm** [tomojitakasu/RTKLIB#650]
Hello:

I am using the 2.4.3 b34 ver, when I clicked the button "Google Maps" from Map View, it showed like this
![Issue](https://user-images.githubusercontent.com/97714622/161503998-7244cc53-3f2b-45a9-95a7-838dbfdafe6e.jpg)
Then, I've entered the API key under the "Google Maps API Key" in the Map View Options. It showed "Something went wrong". How can I solve this problem? Should I enter my API key only or I need to input something else under the "Google Maps API Key" section? And I have done the regedit part already.

**Comment by tomojitakasu** [comment]
See No.139.

http://www.rtklib.com/rtklib_support.htm

**Comment by Leungholammmm** [comment]
Yes I have followed the steps shown on No.139 already but seems not working.

**Comment by tomojitakasu** [comment]
Your API key may be wrong. Test the API key in some Google Maps APs on your Web browser.
